lib/rules/no-mixed-operators.js

```diff
@@ -67,6 +67,18 @@ function includesBothInAGroup(groups, left, right) {
     return groups.some(group => group.indexOf(left) !== -1 && group.indexOf(right) !== -1);
 }
 
+/**
+ * Checks whether the given node's parent is a conditional expression and returns the test node else the left node.
```

Comment:
Is this accurate? It doesn’t look like it checks the parent of the node that’s given as a parameter.

Reply:
Yes. that is wrong. Pushed the new changes

Reply:
Yes. that is wrong. Pushed the new changes